server/activefiles.go

```diff
@@ -220,109 +220,109 @@ func (self *ActiveFileManager) GetReaderForFileName(fileName string) FileReader
 		return nil
 	}
 
-	return activeFile.GetReader(self.fileStore)
+	return activeFile.GetReader(activeFileManager.fileStore)
 }
 
 // This will block until activeFile.currentUpload is set and the file writer has been created
-func (self *ActiveFile) GetReader(fileStore FileStore) FileReader {
-	self.readLocker.Lock()
+func (activeFile *activeFile) GetReader(fileStore fileStore) fileReader {
```

Comment:
Same. `af` or `f` would be better (in the long run, it might take getting used to).